### PR TITLE
Remove Moped support

### LINKED ISSUE #826: Remove moped integration
The [moped gem](https://rubygems.org/gems/moped) is no longer being actively worked on. The last version came out in 2015, and as far as we understand it, the [mongo gem](https://rubygems.org/gems/mongo) is now recommended instead.

Let's remove the moped integration in accordance with [our policy around older packages](https://github.com/appsignal/integration-guide/pull/133) / https://github.com/appsignal/integration-guide/blob/main/maintain/version-support.md#packages.

## To do

- [x] Remove moped integration
- [x] Remove moped event formatter

### PR DESCRIPTION
**Pull request by luismiramirez**
We've decided to not support Moped anymore as it's been 7 years without
maintenance, and it's not the official Ruby Mongo driver anymore.

~~Pending: Update docs info about Moped.~~

Fixes #826

### CODE REVIEW

File: .changesets/remove-moped-support.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
+type: "remove"
+---
+
+Remove Moped support as it is no longer the official ruby Mongo driver and it's been 7 years
+without mainteinance.
```
Comment (unflxw) [inline .changesets/remove-moped-support.md:7]:
```suggestion
Remove Moped support as it is no longer the official Ruby Mongo driver and it's been unmaintained for 7 years.
```

File: .changesets/remove-moped-support.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
```
Comment (unflxw) [inline .changesets/remove-moped-support.md:2]: If we're dropping support for something, maybe it should not be a `patch` release? @tom thoughts?

**Review by jeffkreeftmeijer — APPROVED** [review]
Not sure about the level here. One could call this a patch, as we’re mostly cleaning up old code. However, it could also be a minor release, as we’re removing support for Moped. That depends on if people still use it, I guess. I’m fine with both, but would probably opt for a patch level release here (as I’m assuming Moped’s no longer used).

File: .changesets/remove-moped-support.md (hunk 2 above)
Comment (tombruijn) [reply .changesets/remove-moped-support.md:2]: It should be at least a minor release like Jeff says. I wouldn't want to make this a major version release.

**Review by tombruijn — APPROVED** [review]

File: .changesets/remove-moped-support.md (hunk 2 above)
Comment (luismiramirez) [reply .changesets/remove-moped-support.md:2]: https://docs.appsignal.com/ruby/integrations/mongodb.html#older-mongoid-versions-and-mongomapper I think we virtually removed Moped support in the past. This PR removes the last trace of it left; that's why I chose to go with a patch.

**Pull request merged**